notebooks/thoth_support_github_data.ipynb

```diff
@@ -37,6 +37,7 @@
     "from s3_communication import S3Communication\n",
```

Comment:
<p>How are these conditions / definitions determined? From the thoth team?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Yes.